### PR TITLE
Go version fix

### PR DESCRIPTION
**Pull request by qubelabsio**
Changing the output of go version.

### CODE REVIEW

**Comment by jcstein** [comment]
LGTM!

**Review by jcstein — COMMENTED** [review]
LGTM!

**Comment by YazzyYaz** [comment]
@evan-forbes will the Mamaki upgrade require a golang ver upgrade to 1.19?

**Review by jcstein — APPROVED** [review]
lgtm - will wait for Yaz to merge this one

**Comment by evan-forbes** [comment]
> will the Mamaki upgrade require a golang ver upgrade to 1.19?

currently we're using the same version of go as the cosmos-sdk, which is go 1.18. We've discussed how this is annoying for using not using a go version manager since celestia-node using 1.19, and will likely just change it to 1.19 as its very likely fine.

File: docs/developers/environment.md [hunk 1]
```diff
@@ -66,5 +66,5 @@ go version
 The output should be the version installed:
 
 ```sh
-go version go1.18.2 linux/amd64
+go version go1.19 linux/amd64
```
Comment (YazzyYaz) [inline docs/developers/environment.md:69]: can you change it to `go1.19.1`

Comment (qubelabsio) [reply docs/developers/environment.md]: Done @YazzyYaz

**Pull request merged**